aspnetcore/publishing/iis.md

```diff
@@ -207,7 +207,36 @@ The data protection system has limited support for setting default [machine-wide
 
 ## Configuration of sub-applications
 
-When adding applications to an IIS Site's root application, the root application *web.config* file should include the `<handlers>` section, which adds the ASP.NET Core Module as a handler for the app. Applications added to the root application shouldn't include the `<handlers>` section. If you repeat the `<handlers>` section in a sub-application's *web.config* file, you will receive a 500.19 (Internal Server Error) referencing the faulty config file when you attempt to browse the sub-application.
+Applications added to a root application shouldn't include the ASP.NET Core Module as a handler. If you add the module as a handler in a sub-application's *web.config* file, you receive a 500.19 (Internal Server Error) referencing the faulty config file when you attempt to browse the sub app. Use the following *web.config* for an ASP.NET Core sub app:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <system.webServer>
+    <aspNetCore processPath="%LAUNCHER_PATH%" 
```

Comment:
The env vars are a little weird. They are only in the dev version, they're replaced in the published output. As this is a publish doc we should show published output. E.g. dotnet.exe and an assembly name.